python/Makefile

```diff
@@ -11,3 +11,12 @@ test:
 integtest:
 	pytest -m 'integration' $(PYTEST_ARGS) python/tests
 .PHONY: integtest
+
+format:
+	black python && isort python && cargo fmt
+.PHONY: format
+
+lint:
+	black --check python && isort --check-only python && ruff python && \
+	cargo fmt -- --check && cargo clippy -- -D warnings
+.PHONY: lint
```

Comment:
This will come in handy.  Do you want to add a note to the developers guide?

Reply:
Done.